### DISCUSSION TITLE
RFC on System for Providing Voting Delegations for RING Stakers

### DISCUSSION THREAD

**Discussion by 2xJDubs** [darwinia-network/darwinia#1552] [Ideas]
Currently it is not possible for people staking RING to vote with their staked RING.  The only option is to unstake RING and use it to vote but this incurs an opportunity cost of lost staking rewards while the vote is going and for the locking period.  Currently due to this restriction many votes require the team to provide the amount of Ayes necessary for anything to pass.    The Darwinia team is currently looking into and assessing other options to allow stakers to vote but until that is completed I wanted to offer an idea to allow for people to vote through fund delegation.  With the release of DIP-7 the team is crystalizing on a plan for allowing stakers to vote in the future but it is not clear how long it will take to implement this DIP and if it is expected to be completed soon than this idea can be ignored..

The idea would be to request 20M RING from the Treasury to a multi-sig account that is managed by Partnership Dude, Laki, Saso and myself.  This multi-sig will then be used to delegate RING voting power to people that request it to match what they have staked.  This request will need to be made through an on-chain remark showing that the person controls the address and a manual evaluation of the total amount staked will need to be done.  This delegation system will last until the new version of democracy is implemented that will allow staked RING to vote.  Once this occurs the 20M RING will be returned to the Treasury. 

The process for requesting delegations will be managed through a Google Form where the person will provide an address and the txn for the required on-chain remark showing ownership.  From there someone from the multi-sig will verify how much that person has staked and deposited followed by a delegation of that much RING.  If this idea gains traction and looks to be implemented we can work on getting a script built that will check participant accounts daily to ensure they do not sell staked RING after requesting delegation.

**Comment by 2xJDubs** [comment]
Based on feedback from the team, DIP-7 is expected to be implemented next month so this idea so no longer necessary.

**Comment by hackfisher** [comment]
https://github.com/darwinia-network/vote.ringdao.com will support delegation if I understand correctly, cc @hujw77 

Ref: https://github.com/darwinia-network/RingDAO

**Comment by hackfisher** [comment]
Suppose to support in a few days. https://github.com/darwinia-network/DIP-7/pull/31 cc @hujw77
